WebInstall/explainer_same_domain.md

```diff
@@ -72,7 +72,7 @@ When called on the same domain, the **`install` method will trigger/open the pro
 
 ### The `navigator.install` method
 
-To install a web site/app, the site/app would use the promise-based method`navigator.install(<id>[[, install_url], <params>])`. This method will:
+To install a web site/app, the site/app would use the promise-based method `navigator.install(id, install_url, [<params>])`. This method will:
 
 * Resolve when an installation was completed.
     * The success value will be an object that contains:
```

Comment:
Can't add a comment lower, so putting it here.  I encourage you to open an Issue tracking that there's a single error that is just AbortError.  I think it would be helpful to devs if there was at least one other error that helped identify that the problem was that a `id` was either not found, or the one specified in the call didn't match the one in the target manifest.

Reply:
I believe this is the issue @amandabaker opened for this. https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/898

So far we haven't gotten any external feedback from developers on this, but I think as well the OT will flesh out any additional needs for errors, if any.